packages/edit-site/src/components/navigation-sidebar/navigation-panel/use-debounced-search.js

```diff
@@ -32,13 +32,8 @@ export default function useDebouncedSearch() {
 	const onSearch = useCallback(
 		( value ) => {
 			setSearch( value );
-
-			if ( value.length >= 2 ) {
-				debouncedSetSearchQuery( value );
-				setIsDebouncing( true );
-			} else if ( value.length <= 1 ) {
-				setSearchQuery( '' );
-			}
+			debouncedSetSearchQuery( value );
+			setIsDebouncing( true );
```

Comment:
I changed the implementation to filter on the client-side. Would it make sense to leave denounce and `Loading...` text in the implementation or remove them as well?

Reply:
I think we can remove `Loading`, as I would expect the results to change near-instantly.
The debounce seems short enough to not be of hindrance, while preventing triggering the search mid-type: I'd keep that.